code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -419,7 +419,7 @@
 	item_state = "labcoat"
 	body_parts_covered = CHEST|GROIN|ARMS
 	cold_protection = CHEST|GROIN|ARMS
-	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	min_cold_protection_temperature = 5

```

Comment:
Alright, yeah I couldn't test this so i just modified it to something that seemed mabye ok. I'll fix it